### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by felipemonroy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,27 +36,31 @@ To perform the instalation process you need to have git and docker installed on
...
 * Numeric: The Numeric section gives details on several statistics for each of the numeric columns, as well as an histogram.
 
-* Text: The Text section gives details on the number of missing and unique values corresponding for each text column. Addionally, it specifies the number of rows with empty, only-uppercase and only-lowercase values, finalising with a bar chart of the frequencies.
+* Text: The Text section gives details on the number of missing and unique values corresponding for each text column. Additionally, it specifies the number of rows with empty, only-uppercase and only-lowercase values, finalising with a bar chart of the frequencies.
 
-* Date: The Date section give additional details about the DateTime columns, like the number of week and weekend days for each. Additionaly, it display a bar chart with the frequency of each date.
+* Date: The Date section give additional details about the DateTime columns, like the number of week and weekend days for each. Additionally, it displays a bar chart with the frequency of each date.
 
 ## Test the app
 A series of tests were created to ensure app robustesnest. To run the tests follow the next steps:
```
Comment (javidlc) [inline README.md:66]: change to robustness

File: README.md [hunk 2]
```diff
@@ -1 +1,78 @@
-data_explorer_webapp
+# Data Explorer Webapp
```
Comment (AnaMJaimeR) [inline README.md:1]: Web App, for consistency with the other documentation

**Review by javidlc — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,78 @@
-data_explorer_webapp
+# Data Explorer Webapp 
+[![GitHub release](https://img.shields.io/github/release/AnaMJaimeR/data_explorer_webapp.svg)](https://GitHub.com/AnaMJaimeR/data_explorer_webapp/releases/)
+
+[![Docker](https://badgen.net/badge/icon/docker?icon=docker&label)](https://docker.com/)
+[![Python 3.8.2](https://img.shields.io/badge/python-3.8.2-blue.svg)](https://www.python.org/downloads/release/python-382/)
+
+## Overview
+The Data Explorer Web App is a Python containerised web application, built employing the Streamlit framework. This application allows the user to upload a CSV file to subsequently run a variety of calculations and methods on the inputted data set, and return tables and visuals that are ready to be analysed. All in, the provided outputs aim to help the user to get a deeper understanding on the uploaded data. 
+
```
Comment (AnaMJaimeR) [inline README.md:9]: Please update

File: README.md [hunk 4]
```diff
@@ -1 +1,78 @@
...
+    git clone https://github.com/AnaMJaimeR/data_explorer_webapp.git
+    ```
+
+2. cd into the project root folder
+
+    ```bash
+    cd data_explorer_webapp
+    ```
+
+3. Build Streamlit image
```
Comment (AnaMJaimeR) [inline README.md:25]: the*

File: README.md [hunk 5]
```diff
@@ -1 +1,78 @@
...
+
+4. Create docker container and run
+
+    ```bash
+    docker run --rm -it -p 8501:8501 streamlit:latest
+    ```
+
+## Run the app
+
+After running the container, go to **http://172.17.0.2:8501** to see the app.
```
Comment (AnaMJaimeR) [inline README.md:39]: replace with http://localhost:8501/

Comment (AnaMJaimeR) [reply README.md:39]: to use the app?

File: README.md [hunk 6]
```diff
@@ -1 +1,78 @@
...
+
+## App usage
```
Comment (AnaMJaimeR) [inline README.md:41]: App interactivity?

File: README.md [hunk 7]
```diff
@@ -1 +1,78 @@
...
+* Numeric: The Numeric section gives details on several statistics for each of the numeric columns, as well as an histogram.
+
+* Text: The Text section gives details on the number of missing and unique values corresponding for each text column. Additionally, it specifies the number of rows with empty, only-uppercase and only-lowercase values, finalising with a bar chart of the frequencies.
+
+* Date: The Date section give additional details about the DateTime columns, like the number of week and weekend days for each. Additionally, it displays a bar chart with the frequency of each date.
+
+## Test the app
+A series of tests were created to ensure app robustness. To run the tests follow the next steps:
+
+1. If the Streamlit image is not created
```
Comment (AnaMJaimeR) [inline README.md:68]: has not been created

**Pull request merged**